internal/slicer/report_test.go

```diff
@@ -0,0 +1,214 @@
+package slicer_test
+
+import (
+	"io/fs"
+
+	. "gopkg.in/check.v1"
+
+	"github.com/canonical/chisel/internal/fsutil"
+	"github.com/canonical/chisel/internal/setup"
+	"github.com/canonical/chisel/internal/slicer"
+)
+
+var oneSlice = &setup.Slice{
+	Package:   "base-files",
+	Name:      "my-slice",
+	Essential: nil,
+	Contents:  nil,
+	Scripts:   setup.SliceScripts{},
+}
+
+var otherSlice = &setup.Slice{
+	Package:   "base-files",
+	Name:      "other-slice",
+	Essential: nil,
+	Contents:  nil,
+	Scripts:   setup.SliceScripts{},
+}
+
+var sampleDir = fsutil.Info{
+	Path: "/root/exampleDir",
```

Comment:
/root => /report

/root is the traditional home of the "root" user, so slightly confusing in this context.

Reply:
Agree but this is not the report root but rather the root of the filesystem that we are creating, the report can only contain entries inside it. Maybe `fsroot`? I could also leave it empty `/`.

Reply:
Agreed offline to use `/base`.